Server/src/routes/getBasicDataRouter.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response, Router } from 'express';

```

Comment:
Why are we calling this getBasicDataRouter? Is there a reason for basic?

Reply:
It was to differentiate it from another service, at the time named getDataRouter, which has since been renamed. In truth, I don't know what a good name for this would be, this one was the first one that came to mind. I'm open to suggestions.